src/contracts/interfaces/IEigenPod.sol

```diff
@@ -38,23 +37,28 @@ interface IEigenPod {
         uint64 partialWithdrawalAmountGwei;
     }
 
+    struct ValidatorInfo {
+        // index of the validator in the beacon chain
+        uint64 validatorIndex;
+        // amount of beacon chain ETH restaked on EigenLayer in gwei
+        uint64 restakedBalanceGwei;
+        // status of the validator
+        VALIDATOR_STATUS status;
```

Comment:
should probably move status to the end so if more fields are added it will still pack nicely ?

Reply:
fixed